File: Hebergement/entities/VIP.php
<?php

class VIP
{
  private $_id = null;
  private $_nom = null;
  private $_prenom = null;
  private $_competition = null;
  private $_film = null;

  public function __construct($id, $nom, $prenom, $competition, $film)
  {
    $this->_id = $id;
    $this->_nom = $nom;
    $this->_prenom = $prenom;
    $this->_competition = $competition;
    $this->_film = $film;
  }

  public function getId()
  { return $this->_id; }

  public function getNom()
  { return $this->_nom; }

  public function getPrenom()
  { return $this->_prenom; }

  public function getCompetition()
  { return $this->_competition; }

  public function setCompetition($competition)
  { $this->_competition = $competition; }

  public function getFilm()
  { return $this->_film; }

  public function setFilm($film)
  { $this->_film = $film; }
}

?>

File: Planification/src/entities/Vip.java
package entities;

public class Vip {
    private int id;
    private String nom;
    private String prenom;
    private int idCompetition;
    private int idFilm;
    
    public Vip(int id, String nom, String prenom, int idCompetition, int idFilm)// Permet de stocker les informations sur un VIP
    {
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.idCompetition = idCompetition;
        this.idFilm = idFilm;
    }
    
    public int getId()
    {
        return this.id;
    }
    
    public String getNom()
    {
        return this.nom;
    }
    
    public String getPrenom()
    {
        return this.prenom;
    }
    
    public int getIdCompetition()
    {
        return this.idCompetition;
    }
    
    public int getIdFilm()
    {
        return this.idFilm;
    }   
}
